pkgs/applications/misc/epr/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, python3Packages, fetchFromGitHub, ncurses }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "epr";
+  version = "2.3.0b";
+
+  src = fetchFromGitHub {
+    owner = "wustho";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1a6md3015284hzmx0sby5kl59p7lwv73sq7sid35vrr15zrl0aw7";
+  };
+
+  meta = with lib; {
+    description = "CLI Epub Reader";
+    homepage = "https://github.com/wustho/epr";
+    license = licenses.mit;
+    maintainers = [ ];
```

Comment:
hi, can you add yourself as a maintainer?

Reply:
Would be great, but I already have a PR adding me to the maintainer list (for `ruplacer`). Is it okay for me to duplicate these changes here?

Reply:
Either that or wait until ruplacer is merged in, both are fine